AwfulCore/Sources/AwfulCore/Model/Post.swift

```diff
@@ -27,6 +27,9 @@ public class Post: AwfulManagedObject, Managed {
     /// When the post appeared.
     @NSManaged public var postDate: Date?
     
+    /// Raw post date straight from the forums
+    @NSManaged public var postDateRaw: String
```

Comment:
This is marked `optional` in the `.xcdatamodel` so it should be `String?` here, I think?

Alternately, ok to default to empty string in the model and leave this as-is. But one or the other.

Reply:
Made it optional!